aquadoggo/src/config.rs

```diff
@@ -19,6 +19,8 @@ const DEFAULT_SQLITE_NAME: &str = "aquadoggo-node.sqlite3";
 /// Blobs directory
 pub const BLOBS_DIR_NAME: &str = "blobs";
 
+pub const BLOBS_SYMLINK_DIR_NAME: &str = "documents";
```

Comment:
Is this a good name? (and give it a comment)

Reply:
Would be nice if we could get rid of this extra path and use our own router to serve static files.

I would think of something like this:

Route: `/blobs/:documentId/?:documentViewId`

With these two parameters `documentId` and optionally `documentViewId` we would resolve the right file on the filesystem and return the bytes to the HTTP client. Additionally we would also do a database lookup, finding out the MIME type and adding that to the HTTP response.

Reply:
Yep, will add a note and tackle this in a separate PR.